2010-10-10  Michael Tautschnig  <mt@debian.org>

	* install_packages: Fix dpkg hold output (thanks michael <mmlogin@gmail.com>).
Index: trunk/bin/install_packages
===================================================================
--- trunk.orig/bin/install_packages
+++ trunk/bin/install_packages	
@@ -184,7 +184,7 @@
   my $packlist = join(' ',@{$list{$atype}});
 
   if ($atype eq "hold") {
-    my $hold = join " hold\n", @{$list{hold}}, " hold\n";
+    my $hold = join " hold\n", @{$list{hold}}, "";
     execute("echo \"$hold\" | $rootcmd $command{hold}");
     next;
   }
